_posts/2019-05-01-galactic-news.md

```diff
@@ -1,7 +1,9 @@
 ---
 site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
 title: Galactic News
-tags: [galaxy]
+tags: 
+- galaxy
+- newsletter
```

Comment:
newletter? Can we have a list of tags in the README of this repo ... at least the most common ones.

Reply:
done